docs/source/using/addons.adoc

```diff
@@ -100,53 +95,52 @@ $ openshift admin ca create-server-cert \
   --key=/etc/secrets/registry.key
 ----
 
-<<table-supported-addon-variables>> shows the currently supported variables
-which are available for interpolation.
-
 [[table-supported-addon-variables]]
 .Supported add-on variables
+
```

Comment:
What do you mean "not good form"? I didn't change the heading of the table or the structure, I only removed the unnecessary definite articles, the text is clear without them and there's no need for "The" in the beginning of every line.